src/models/PurchasedAlbum.js

```diff
@@ -0,0 +1,10 @@
+const mongoose = require('mongoose');
+
+const PurchasedAlbumSchema = new mongoose.Schema({
+  albumId: { type: Schema.Types.ObjectId, ref: 'Album', required: true },
+  userId: { type: Schema.Types.ObjectId, ref: 'User', required: true },
```

Comment:
```suggestion
  albumId: { type: mongoose.Schema.Types.ObjectId, ref: 'Album', required: true },
  userId: { type: mongoose.Schema.Types.ObjectId, ref: 'User', required: true },
```

Reply:
Otherwise we get the error `Uncaught ReferenceError: Schema is not defined`.

Other option would be to add `const { Schema } = require('mongoose');` to the top of the file to import that variable directly